CMakeLists.txt

```diff
@@ -574,3 +575,284 @@ endfunction()
 Excitation_Funcs()
 Integrated_vn()
 Hydro_Lifetimes()
+
+
+################################################################################
+#
+# Add functionality to use any custom SMASH config for the initial conditions
+#
+################################################################################
+function(run_custom_hybrid
+        )
+  message(STATUS "Configuring hybrid run for custom IC config.")
+  set(results_folder "${CMAKE_CURRENT_BINARY_DIR}/Hybrid_Results/Custom")
+  file(MAKE_DIRECTORY ${results_folder})
+
+  set(smash_IC_files "")
+  set(vhlle_freezeout_hypersurfaces "")
+  set(sampled_particle_lists "")
+  set(final_particle_lists "")
+  set(all_analysis_outputs "")
+  set(all_plots "")
+
+  set(num_runs "100")
```

Comment:
Because it's two separate functions in the CMakeLists.txt. One is the `run_one_energy`function, which runs the hybrid at a specific predefined energy, and the other function is the `run_custom_hybrid`function. Both rely on the same idea/functionality but needed to be implemented separately to have a specific target for the custom version.

Reply:
As a result, `num_runs`that is declared in `run_one_energy` is not available in `run_custom_hybrid`, so it needed to be declared again in that scope.

Reply:
Ok I see